### PR TITLE
Updated conf.py for sphinx and removed layout.html

### PR DESCRIPTION
**Pull request by MrTheodor**
Removed layout.html, it looks like as a template for home page but there are no other files.

updated conf.py to fetch automaticly version of ESPP from Version() module

copied content of http://www.espresso-pp.de/

cc: @stuehn @junghans

### CODE REVIEW

File: doc/ug/conf.py [hunk 1]
```diff
@@ -12,6 +12,16 @@
 # serve to show the default.
 
 import sys, os
+import datetime
+
+# Gets version directly from the code
+try:
+    import espressopp
```
Comment (junghans) [inline doc/ug/conf.py]: Not sure if that will even use the current version or the one which is installed.
I might be better to turn `conf.py` into a cmake configure_file and replace the version at cmake time.

Comment (MrTheodor) [inline doc/ug/conf.py]: Would it be possible? The version is hardcoded in Version.hpp so some sort of parsing would be required. Other thing is that, during the building of documentation, ```espressopp``` module has to be available as ```autodoc``` depends on it.

Comment (junghans) [inline doc/ug/conf.py]: I see, in #43 we could make the `ug` target depend on the espresso++ model and set the environment correctly like you suggested.

**Comment by MrTheodor** [comment]
Included in #43

**Comment by junghans** [comment]
Leave it open until #43 is merged.

**Comment by junghans** [comment]
I think we should leave the layout like it is and make http://espressopp.github.io/ just a copy of http://www.espresso-pp.de/ with more recent content on it.

It shouldn't be hard to generate and auto-deploy the [pdf manual](https://github.com/espressopp/espressopp/blob/master/espp_manual.pdf) as well.

**Comment by MrTheodor** [comment]
Working on that.

cc: @acfogarty

**Pull request closed without merging**